src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2334,7 +2319,8 @@ namespace winrt::TerminalApp::implementation
                 sui.SetHostingWindow(reinterpret_cast<uint64_t>(*_hostingHwnd));
             }
 
-            sui.PreviewKeyDown({ this, &TerminalPage::_SUIPreviewKeyDownHandler });
+            // GH#8767 - let unhandled keys in the SUI try to run commands too.
+            sui.KeyDown({ this, &TerminalPage::_KeyDownHandler });
```

Comment:
FYI: this seems like it was the only difference from #8882. We're doing a `sui.KeyDown` instead of a `sui.PreviewKeyDown`. That's probably why key chords like <kbd>ctrl+c</kbd> were being eaten by us instead of XAML controls (like text boxes).